app/graphql/mutations/user/delete_list.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Mutations
+  module User
+    class DeleteList < AuthBaseMutation
+      type Types::ListType
```

Comment:
This mutation should return `Types::DeleteListType` that has one field (`deleted_list_id`).
Optionally, you can create a general delete type, not related to the specific instance. Smth like `Types::DeleteItemType`. It will have a `deleted_item_id` field